src/main/webapp/WEB-INF/tags/instructor/navBar.tag

```diff
@@ -4,48 +4,48 @@
-                    <a class='nav evaluations' data-link="instructorEval" href="${data.instructorFeedbackSessionsLink}">Sessions</a>
-                </li>
-                <li<c:if test="${fn:contains(data.class,'Student')}"> class="active"</c:if>>
-                    <a class='nav students' data-link="instructorStudent" href="${data.instructorStudentListLink}">Students</a>
-                </li>
-                <li<c:if test="${fn:contains(data.class,'Search')}"> class="active"</c:if>>
-                    <a class='nav search' data-link="instructorSearch" href="${data.instructorSearchLink}">
-                        Search
-                    </a>
-                </li>
-                <li>
-                    <a class="nav help" href="/instructorHelp.jsp" target="_blank" rel="noopener noreferrer">Help</a>
-                </li>
-            </ul>
-            <ul class="nav navbar-nav pull-right">
-                <li><a id="btnLogout" class="nav logout" href="<%= Const.ActionURIs.LOGOUT %>">Logout
+  <div class="container">
+    <div class="navbar-header">
+      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#contentLinks">
+        <span class="sr-only">Toggle navigation</span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+        <span class="icon-bar"></span>
+      </button>
+      <t:teammatesLogo/>
+    </div>
+    <div class="collapse navbar-collapse" id="contentLinks">
+      <ul class="nav navbar-nav">
+        <li<c:if test="${fn:contains(data.class,'Home')}"> class="active"</c:if>>
+          <a class='nav home' data-link="instructorHome" href="${data.instructorHomeLink}">Home</a>
+        </li>
+        <li<c:if test="${fn:contains(data.class,'InstructorCourse') && !fn:contains(data.class, 'CourseStudent')}"> class="active"</c:if>>
+          <a class='nav courses' data-link="instructorCourse" href="${data.instructorCoursesLink}">Courses</a>
+        </li>
+        <li<c:if test="${fn:contains(data.class,'Feedback')}"> class="active"</c:if>>
+          <a class='nav evaluations' data-link="instructorEval" href="${data.instructorFeedbackSessionsLink}">Sessions</a>
+        </li>
+        <li<c:if test="${fn:contains(data.class,'Student')}"> class="active"</c:if>>
+          <a class='nav students' data-link="instructorStudent" href="${data.instructorStudentListLink}">Students</a>
+        </li>
+        <li<c:if test="${fn:contains(data.class,'Search')}"> class="active"</c:if>>
+          <a class='nav search' data-link="instructorSearch" href="${data.instructorSearchLink}">
+            Search
+          </a>
+        </li>
+        <li>
+          <a class="nav help" href="/instructorHelp.jsp" target="_blank" rel="noopener noreferrer">Help</a>
+        </li>
+      </ul>
+      <ul class="nav navbar-nav pull-right">
+        <li><a id="btnLogout" class="nav logout" href="<%= Const.ActionURIs.LOGOUT %>">Logout
```

Comment:
I'd suggest shifting the opening `<a>` tag down to a new line here, even though it was on the same line in the old code.